lib/biran/configurinator.rb

```diff
@@ -33,6 +34,11 @@ def file_tasks
       files_to_generate.keys
     end
 
+    def debug_stuff
+      puts "app env2 is: #{app_env}"
```

Comment:
👏 i assume this wont be in the final PR?

Reply:
Its in the PR description... "not working with debug code"